### PR TITLE
CUMULUS-3798: Merge v18.3.2 release updates into master

### PR DESCRIPTION
**Pull request by paulpilone**
**Summary:** Summary of changes

Addresses [CUMULUS-3798: Create and publish Release 18.3.2](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3798)

## Changes

* Merges v18.3.2 release updates into master.

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by paulpilone** [comment]
This PR merges the final release updates back into master. It's mostly documentation but the tricky part is always the CHANGELOG.

File: tasks/update-granules-cmr-metadata-file-links/package.json [hunk 1]
```diff
@@ -40,15 +40,15 @@
...
+    "@cumulus/aws-client": "18.3.2",
+    "@cumulus/cmrjs": "18.3.2",
+    "@cumulus/common": "18.3.2",
     "@cumulus/cumulus-message-adapter-js": "2.2.0",
-    "@cumulus/distribution-utils": "18.3.1",
+    "@cumulus/distribution-utils": "18.3.2",
     "lodash": "^4.17.15"
   },
   "devDependencies": {
     "@cumulus/aws-client": "18.3.1",
```
Comment (etcart) [inline tasks/update-granules-cmr-metadata-file-links/package.json:51]: should this not also be brought up to 18.3.2?

**Review by etcart — CHANGES_REQUESTED** [review]

File: tasks/update-granules-cmr-metadata-file-links/package.json (hunk 1 above)
Comment (paulpilone) [reply tasks/update-granules-cmr-metadata-file-links/package.json:51]: I suppose it should. Not sure why it's not bumped up as part of the update script.

**Review by etcart — APPROVED** [review]
awesome

**Pull request merged**